tests/integration/tier0/non-destructive/eus/test_eus_support.py

```diff
@@ -106,3 +106,24 @@ def test_eus_support(
                 )
             c2r.expect("Repositories enabled through subscription-manager", timeout=120)
             c2r.sendcontrol("c")
+
+
+@pytest.mark.test_rhsm_eus_account_conversion
+def test_rhsm_non_eus_account(convert2rhel):
```

Comment:
```suggestion
@pytest.mark.test_rhsm_non_eus_account
def test_rhsm_non_eus_account(convert2rhel):
```
Copy&Paste error.
Additionally, this test did not even run, given the lack of metadata in main.fmf

Reply:
<img src="https://media3.giphy.com/media/Ty9Sg8oHghPWg/giphy.gif"/>